spec/features/visitor_checkout_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'As a visitor' do
+  describe 'when I add items to my cart and I visit my cart page' do
+    it 'should see a message telling me to register or login to checkout' do
+      item_1 = create(:item)
+      item_2 = create(:item)
```

Comment:
Super small but I don't think you really need item_2?

Reply:
I'm kind of with  both of you on item_2. You may be strictly right, Anna, but sometimes it just doesn't feel right testing a list with just one item!

Reply:
@asmolentzov and @saleratus  - I see what you mean - I just included it to make sure that item 2 didn't show up. Not super sad-path-y but let me know if you want me to take that out.

Reply:
I'm fine either way, but I don't see any spot where you're testing that item_2 doesn't show up when it shouldn't? Or that that is super necessary here, though I'm always for double-checking. Or is it just so that having another item on the page (presumably) doesn't mess things up somehow?

Reply:
@asmolentzov I see what you mean now - I'm taking this item_2 out as well as the other. Thanks! =]